packages/src/@terra-money/wallet-controller/controller.ts

```diff
@@ -983,6 +983,7 @@ export class WalletController {
         {
           connectType: ConnectType.PLUGINS,
           terraAddress: session.terraAddress || 'not created',
+          metadata: session.getMetadata ? session.getMetadata() : undefined,
```

Comment:
Here I'll do just a minor syntax change:
```suggestion
          metadata: session.getMetadata && session.getMetadata(),
```